sdk/messaging/azeventhubs/internal/amqpwrap/amqpwrap.go

```diff
@@ -3,6 +3,7 @@
 
 // Package amqpwrap has some simple wrappers to make it easier to
 // abstract the go-amqp types.
+
```

Comment:
If there's no space, then the above comment will be used as the doc comment for this package.  Is this intentional?